cmd/liqoctl/cmd/install.go

```diff
@@ -138,8 +138,8 @@ func newInstallCommand(ctx context.Context, f *factory.Factory) *cobra.Command {
 
 	cmd.PersistentFlags().BoolVar(&options.OnlyOutputValues, "only-output-values", false,
 		"Generate the pre-configured values file for further customization, instead of installing Liqo (default false)")
-	cmd.PersistentFlags().StringVar(&options.ValuesPath, "dump-values-path", "./values.yaml",
```

Comment:
I would add a comment to specify that the default value is set after the check if the flag has been set or not